polaris-react/src/components/Divider/Divider.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import type {ShapeBorderWidthScale} from '@shopify/polaris-tokens';
+
+import {Box} from '../Box';
+
+export type BorderTokenAlias =
+  | 'base'
+  | 'dark'
+  | 'divider'
+  | 'divider-on-dark'
+  | 'transparent';
+
+export interface DividerProps {
+  /** Divider style */
+  border?: BorderTokenAlias;
```

Comment:
It shouldn't be called `color` because it's not setting border color but rather border shorthand. So I think `border` is more correct. Or `borderStyle` if we want to be more descriptive.

See https://github.com/Shopify/polaris/blob/main/polaris-tokens/src/token-groups/shape.ts#LL49:LL62